crates/codegen/ebnf/src/builder.rs

```diff
@@ -430,3 +433,13 @@ impl Builder {
         Expression::new_reference(reference.to_string())
     }
 }
+
+enum Terminal {
```

Comment:
These can be easily `Copy` to avoid `&` everywhere

Reply:
Good point.